jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_PREP

```diff
@@ -32,6 +32,9 @@ mkdir -p "${COMOUT}"
 # COMIN_GES and COMIN_GES_ENS are used in script
 export COMIN_GES="${ROTDIR}/${GDUMP}.${gPDY}/${gcyc}/ocean"
 
+# Add UFSDA to PYTHONPATH
+export PYTHONPATH=${HOMEgfs}/sorc/gdas.cd/ush/:${PYTHONPATH}
```

Comment:
Following from [@aerorahul's comment](https://github.com/NOAA-EMC/global-workflow/pull/1106#discussion_r1088207136) on another PR, I think this should be:
```
socaPATH="${HOMEgfs}/sorc/gdas.cd/ush/"
PYTHONPATH="${PYTHONPATH:+${PYTHONPATH}:}${socaPATH}"
export PYTHONPATH
```

Reply:
Sounds good @WalterKolczynski-NOAA . 
@AndrewEichmann-NOAA , replace `socaPATH` by `ufsdaPATH` from @WalterKolczynski-NOAA 's suggestion above.
FYI @WalterKolczynski-NOAA , while I agree with you that `soca` should take over the world, it's also "just" the marine DA.